### DISCUSSION TITLE
[A-L] (2024/25) Foglio 1 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Rurik-D** [sapienzastudentsnetwork/algebra#463] [Esercizi - Viaggi/Cherubini]
![image](https://github.com/user-attachments/assets/ec9484a2-431c-4f61-9180-ecc4dcb6fbbc)

**Comment by Rurik-D** [comment]
![image](https://github.com/user-attachments/assets/532a61d8-6d5a-4579-af8e-6ced9faab141)

**Comment by niccolo-fato** [comment]
![photo_2024-09-29_17-38-43](https://github.com/user-attachments/assets/6e90baf0-9aec-42e1-8ad5-98a9b144e7b9)
